src/fundus/parser/utility.py

```diff
@@ -542,8 +542,14 @@ def get_versions_from_node(
                 query_width = f"{param}:{value}"
 
     # get width, height and init calculator
-    width = float(source.get("width") or 0) or None
-    height = float(source.get("height") or 0) or None
+    try:
+        width = float(source.get("width") or 0) or None
+    except ValueError:
+        width = None
+    try:
+        height = float(source.get("height") or 0) or None
+    except ValueError:
+        height = None
```

Comment:
Do you know which case leads to a `ValueError`? In general, I would like to avoid `try ... except` blocks, especially in highly frequented code.